releasenotes/notes/single-span-sampling-tags-fix-235b289006f2c635.yaml

```diff
@@ -0,0 +1,3 @@
+fixes:
+  - |
+    tracing: This fix removes single-span-sampling tags from span contexts in sampled traces, in which case they are redundant
```

Comment:
I'll let someone else comment to confirm, but we probably don't need a release note here since this is an internal implementation detail, so nothing should change for users.

meaning we can likely change the PR type from `fix:` to `chore(internal):`

Reply:
How do I stop this check from failing? https://github.com/DataDog/dd-trace-py/actions/runs/3911832547/jobs/6685680025